docs/how-to-upload-manifest.md

```diff
@@ -110,9 +95,18 @@ Each release of an operator manifest is considered immutable. If you try to push
 }
 ```
 
-If you change your operator manifest then bump up the version in `RELEASE` before pushing the manifest.
+If you change your operator manifest, you should always bump up the version of the version/release before pushing the manifest.
 
 ### Delete Operator Manifest
+
 *You will need `admin` privileges to delete repositories.*
 
-For now, we have been deleting manifest(s) manually by going to `quay.io`. Go to the `Settings` tab of your desired repository and click the `Delete Application` button. 
\ No newline at end of file
+For now, we have been deleting manifest(s) manually by going to `quay.io`. Go to the `Settings` tab of your desired repository and click the `Delete Application` button.
+
+### Creating New Manifests
+
+By default, when you push a new manifest to `quay.io`, the repository will be private. Ensure that you make it public for it to be visible to the marketplace operator.
+
+### Updating the `OperatorSource`
```

Comment:
This section should go into the main README and not here. It should be done in a separate PR.